ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1,1345 +1,1350 @@
+EventType,string,Mandatory,Authentication,Enumerated,Logon|Logoff|Elevate,
+EventType,string,Mandatory,Common,,,
+EventType,string,Mandatory,Dhcp,Enumerated,Assign|Renew|Release|DNS Update|,
+EventType,string,Mandatory,Dns,Enumerated,Query|Status|Notify|Update|DNS Stateful Operations,
+EventType,string,Mandatory,FileEvent,Enumerated,FileAccessed|FileCreated|FileModified|FileDeleted|FileRenamed|FileCopied|FileMoved|FolderCreated|FolderDeleted|FolderMoved|FolderCopied|FolderModified|FileCreatedOrModified|FileAttributesUpdated,
+EventType,string,Mandatory,NetworkSession,Enumerated,NetworkSession|L2NetworkSession|EndpointNetworkSession|Flow,
+EventType,string,Mandatory,ProcessEvent,Enumerated,ProcessCreated|ProcessTerminated,
+EventType,string,Mandatory,RegistryEvent,Enumerated,RegistryKeyCreated|RegistryKeyDeleted|RegistryKeyRenamed|RegistryValueDeleted|RegistryValueSet,
+EventType,string,Mandatory,UserManagement,Enumerated,UserCreated|UserDeleted|UserModified|UserLocked|UserUnlocked|UserDisabled|UserEnabled|PasswordChanged|PasswordReset|GroupCreated|GroupDeleted|GroupModified|UserAddedToGroup|UserRemovedFromGroup|GroupEnumerated|UserRead|GroupRead,
+EventType,string,Mandatory,WebSession,Enumerated,HTTPsession|WebServerSession|ApiRequest,
+EventUid,string,Recommended,AuditEvent,,,
+EventUid,string,Recommended,Authentication,,,
+EventUid,string,Recommended,Common,,,
+EventUid,string,Recommended,Dhcp,,,
+EventUid,string,Recommended,Dns,,,
+EventUid,string,Recommended,FileEvent,,,
+EventUid,string,Recommended,NetworkSession,,,
+EventUid,string,Recommended,ProcessEvent,,,
+EventUid,string,Recommended,RegistryEvent,,,
+EventUid,string,Recommended,UserManagement,,,
+EventUid,string,Recommended,WebSession,,,
+EventVendor,string,Mandatory,AuditEvent,Enumerated,Microsoft|AWS|Barracuda|Cisco|Dataminr|Vectra,
+EventVendor,string,Mandatory,Authentication,Enumerated,Salesforce|AWS|Barracuda|Cisco|Microsoft|Okta|PostgreSQL|OpenBSD|Linux|Vectra|SentinelOne,
+EventVendor,string,Mandatory,Common,,,
+EventVendor,string,Mandatory,Dhcp,,,
+EventVendor,string,Mandatory,Dns,Enumerated,Cisco|Corelight|GCP|Infoblox|Microsoft|Zscaler|Vectra AI|SentinelOne,
+EventVendor,string,Mandatory,FileEvent,Enumerated,Microsoft|SentinelOne,
+EventVendor,string,Mandatory,Dns,Enumerated,Cisco|Corelight|GCP|Infoblox|Microsoft|Zscaler|Vectra AI,
+EventVendor,string,Mandatory,FileEvent,Enumerated,Microsoft|SentinelOne,
+EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Barracuda|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|Forcepoint|Cynerio|SentinelOne|CrowdStrike|SonicWall,
+EventVendor,string,Mandatory,FileEvent,Enumerated,Microsoft|SentinelOne|VMware,
+EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Barracuda|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|Forcepoint|Cynerio|SentinelOne|VMware|SonicWall,

```

Comment:
@jaimeesc can you please check why this file showing too many change, making it difficult to see what's changing. Ideally, only one line should show as change i.e. addition of SonicWall in EventVendor for NetworkSession.

Reply:
I'll see if I can figure that out. I had a couple of other files that also appear to have way more changes than I actually made.

Reply:
I rolled back the file, committed it, then made the edits again and committed it.